TSTutorial/homeworks/hw_4/Order.ts

```diff
@@ -0,0 +1,65 @@
+import {TOrder, IMeal} from "./Types"
+import {OrderStatusEnum, PaymentMethodEnum} from "./Enums";
+
+export class Order {
```

Comment:
update this class to be used in OrderService, and implement interface for services like OrderService to depend on